.github/workflows/community.yml

```diff
@@ -0,0 +1,19 @@
+name: Community
+on:
+  issues:
+    types: [opened, edited, milestoned]
+  pull_request:
+    types: [opened, syncronized, ready_for_review, unlabeled]
```

Comment:
- `syncronized` doesn't seem to be a valid event type. I guess you meant `synchronize`? (https://help.github.com/en/actions/reference/events-that-trigger-workflows). Though I am not sure what exactly the `synchronize` event corresponds to.
- Also, I think you can add `reopened` (after it was previously closed)  to the event list. 
- And I am not sure if including `unlabeled` makes sense. Does that mean that if I would remove the `automerge` label from my PR, this workflow would get triggered again and add it again?

Reply:
oh je!
1- sorry, extra 'd' :sweat_smile: It’s to run the workflow everytime the PR needs to be syncronized so for example, if pushed new commits.
2-  like it. Done in ccdb3d328b472f007adf1f2c27c4437690d43617
3- that was the idea, and why I added it, but after trying came to the conclusion that when the PR had already a label and you manually remove it, it won't retriger again. So removed.

Reply:
If it's triggered for every commit I am not sure if that would be really necessary.

Reply:
me neither, tbh, the bot doesn't push to the branch. I am gonna remove it. 🤔 
Now it fails because it tries to label that PR that is already labeled, so I would need to remove the label by hand before retrigger.